src/ControlSystem/Tags.hpp

```diff
@@ -46,6 +46,16 @@ struct ControlSystemInputs {
   static std::string name() { return ControlSystem::name(); }
   using group = ControlSystemGroup;
 };
+
+/// \ingroup OptionTagsGroup
+/// \ingroup ControlSystemGroup
+/// Option tag on whether to write data to disk.
+struct WriteDataToDisk {
```

Comment:
You removed "data" from the name of a function on @nilsdeppe's request, but here it is still. How about `OutputX`, where `X` gives some indication what is actually written out? E.g. `OutputDiagnostics` if it's debug info; other ideas I have are `OutputFunctionsOfTime`, `OutputControlSignal`.

With the current name I find it unclear if I have to expect heaps of volume data, or only a row of doubles per timestep. If the option name can't make this clear, please expand the help string a little.

If you only write a row of doubles, would it be reasonable to just always write this data out, without the switch to disable it?